spec/models/miq_expression_spec.rb

```diff
@@ -1631,4 +1631,100 @@
       end
     end
   end
+
+  describe ".get_col_operators" do
+    subject { described_class.get_col_operators(@field) }
+
+    it "returns array of available operations if parameter is :count" do
+      @field = :count
+      expect(subject).to eq(["=", "!=", "<", "<=", ">=", ">"])
+    end
+
+    it "returns list of available operations if parameter is :regkey" do
+      @field = :regkey
+      expect(subject).to eq(["=",
+                             "STARTS WITH",
+                             "ENDS WITH",
+                             "INCLUDES",
+                             "IS NULL",
+                             "IS NOT NULL",
+                             "IS EMPTY",
+                             "IS NOT EMPTY",
+                             "REGULAR EXPRESSION MATCHES",
+                             "REGULAR EXPRESSION DOES NOT MATCH",
+                             "KEY EXISTS",
+                             "VALUE EXISTS"])
+    end
+
+    it "returns list of available operations for field type 'string'" do
+      @field = "ManageIQ::Providers::InfraManager::Vm.advanced_settings-name"
+      expect(subject).to eq(["=",
+                             "STARTS WITH",
+                             "ENDS WITH",
+                             "INCLUDES",
+                             "IS NULL",
+                             "IS NOT NULL",
+                             "IS EMPTY",
+                             "IS NOT EMPTY",
+                             "REGULAR EXPRESSION MATCHES",
+                             "REGULAR EXPRESSION DOES NOT MATCH"])
+    end
+
+    it "returns list of available operations for field type 'integer'" do
+      @field = "ManageIQ::Providers::InfraManager::Vm-cpu_limit"
+      expect(subject).to eq(["=", "!=", "<", "<=", ">=", ">", "RUBY"])
+    end
+
+    it "returns list of available operations for field type 'float'" do
+      @field = "Storage-v_provisioned_percent_of_total"
+      expect(subject).to eq(["=", "!=", "<", "<=", ">=", ">", "RUBY"])
+    end
+
+=begin
```

Comment:
Could that go back to MySQL, or an older version of Rails perhaps? Would be great to delete it if it's not used, but OK to do that in a follow up PR